riscv/encoding.h

```diff
@@ -709,6 +709,10 @@
 #define MASK_CTZ 0xfff0707f
 #define MATCH_CTZW 0x6010101b
 #define MASK_CTZW 0xfff0707f
+#define MATCH_CZERO_EQZ 0xe005033
+#define MASK_CZERO_EQZ 0xfe00707f
+#define MATCH_CZERO_NEZ 0xe007033
+#define MASK_CZERO_NEZ 0xfe00707f
```

Comment:
This `encoding.h` file is generated from the riscv-opcodes project. It should not be hand-edited. We should see the riscv-opcodes commit hash at the top of this file change to match the updated source, and it should be done as a separate commit in this PR.

Reply:
Thanks for catching this.  However, we're in a pickle: see https://github.com/riscv/riscv-opcodes/pull/147

Reply:
Ugly, but you could make a PR on riscv-opcodes that forks from before riscv/riscv-opcodes#147, cherry-picks riscv/riscv-opcodes#152, then merges back to master with ultimately no changes on master. Then we'd have a git hash to build from here.

Reply:
It's preferable to blocking this PR, which is unrelated to the breakage.  I'll do that.